crates/bevy_render/src/camera/camera.rs

```diff
@@ -362,10 +362,10 @@ pub fn camera_system<T: CameraProjection + Component>(
         .collect();
 
     let mut added_cameras = vec![];
-    for entity in &mut queries.p1().iter() {
```

Comment:
lol thats an incredibly fun side effect of `impl<T: Iterator> Iterator for &mut T`